Documentation/intro-to-clr.md

```diff
@@ -140,7 +140,7 @@ For type safety, conceptually each memory allocation is associated with a type.
 
 One the most important of these type-specific guarantees is that the visibility attributes associated with a type (and in particular with fields) are enforced.  Thus, if a field is declared to be private (accessible only by the methods of the type), then that privacy will indeed be respected by all other type-safe code.  For example, a particular type might declare a count field that represents the count of items in a table.  Assuming the fields for the count and the table are private, and assuming that the only code that updates them updates them together, there is now a strong guarantee (across all type-safe code) that the count and the number of items in the table are indeed in sync.  When reasoning about programs, programmers use the concept of type safety all the time, whether they know it or not.  The CLR elevates type-safety from being simply a programming language/compiler convention, to something that can be strictly enforced at run time.
 
-### Enforcing memory safety (Verifiable Code)
+### Enforcing Verifiable Code (Memory and Type Safety)
```

Comment:
"Enforcing Verifiable Code" doesn't quite make sense to me. Verifiable code is something that you arrive at by construction, not by "enforcement". My view (and I think that this also what the next paragraph by @vancem is saying) is that verifiable code is what is used to enforce type safety.

How about "Verifiable Code - Enforcing Memory and Type Safety"??